### PR TITLE
Update changelog for 1.4.3

### PR DESCRIPTION
**Pull request by alexander-schranz**
/cc @dantleech 

[1.4.2...master](https://github.com/jackalope/jackalope-jackrabbit/compare/1.4.2...master)

### CODE REVIEW

**Comment by dantleech** [comment]
i don't maintain this repo typically - but happy to merge and tag with @dbu 's :+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 Changelog
 =========
 
-unreleased
-----------
+1.4.3
+-----
 
 * Support PHP 8.1
```
Comment (dbu) [inline CHANGELOG.md:7]:
```suggestion
* Support PHP 8.1
* Allow installation with Symfony 6
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+* Allow to use another version in jackrabbit.sh script
```
Comment (dbu) [inline CHANGELOG.md:9]:
```suggestion
* Allow to use another jackrabbit version in jackrabbit.sh script by setting the environment variable `JACKRABBIT_VERSION`
```

**Comment by dbu** [comment]
here we go https://github.com/jackalope/jackalope-jackrabbit/releases/tag/1.4.3

**Pull request merged**